server_config.toml

```diff
@@ -150,7 +150,7 @@ listener = "tcp://0.0.0.0:46658"
 # This file needs to be in the root directory
 private_validator_file = "priv_validator.json"
 
-  # Tendermint requires additional configuration parameters.
+  # Tendermint requires additional configuration parameters.TransactAndHold
```

Comment:
This is a typo, but also we should no longer have server_config.toml in the root directory.